frontend/src/initKea.ts

```diff
@@ -123,7 +123,14 @@ export function initKea({
 
     // To enable logging, run localStorage.setItem("ph-kea-debug", true) in the console
     if (window.JS_KEA_VERBOSE_LOGGING || ('localStorage' in window && window.localStorage.getItem('ph-kea-debug'))) {
-        plugins.push(posthogKeaLogger)
+        plugins.push(
+            posthogKeaLogger({
+                logger: (title, stateEvent) => {
+                    const ph: PostHog | undefined = window.posthog
+                    ph?.sessionRecording?.tryAddCustomEvent('state-log', { title, stateEvent })
+                },
+            })
+        )
```

Comment:
should probably provide this as a prebaked option in the posthogKea and posthogRedux loggers since it requires the magic `state-log` string